Civi/Api4/Action/Birthdays/sendGreetings.php

```diff
@@ -0,0 +1,63 @@
+<?php
+/*
+ * Copyright (C) 2022 SYSTOPIA GmbH
+ *
+ *  This program is free software: you can redistribute it and/or modify
+ *  it under the terms of the GNU Affero General Public License as published by
+ *  the Free Software Foundation in version 3.
+ *
+ *  This program is distributed in the hope that it will be useful,
+ *  but WITHOUT ANY WARRANTY; without even the implied warranty of
+ *  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ *  GNU Affero General Public License for more details.
+ *
+ *  You should have received a copy of the GNU Affero General Public License
+ *  along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+namespace Civi\Api4\Action\Birthdays;
+
+use Civi\Api4\Generic\Result;
+use CRM_Birthdays_ExtensionUtil as E;
+
+final class sendGreetings extends \Civi\Api4\Generic\AbstractAction {
+
```

Comment:
There's some code style/formatting/indentation issues - let's be as strict as possible with that, please.